packages/sdk/src/restclient.spec.ts

```diff
@@ -509,10 +510,11 @@ describe("RestClient", () => {
         pendingWithoutCosmos();
 
         // query by one key
-        const model = await client.queryContractRaw(contractAddress!, expectedKey);
-        expect(model).not.toBeNull();
-        expect((model as any).verifier).toBeDefined();
-        expect((model as any).beneficiary).toBeDefined();
+        const raw = await client.queryContractRaw(contractAddress!, expectedKey);
+        expect(raw).not.toBeNull();
+        const model = JSON.parse(fromAscii(raw!));
```

Comment:
You can replace the `expect`ation here to `assert(raw, "Query result must not be falsy")` with `import { assert } from "@iov/utils";`. This helper throws when the expression is falsy and has an amazing new TypeScript signature that makes the static analyzier know that. `raw!` will then become `raw` since it is narrowed down to `Uint8Array`.

Reply:
`asserts condition` ?
https://github.com/iov-one/iov-core/blob/master/packages/iov-utils/src/assert.ts

Time for me to go back to the manual.

I'll make the update

Reply:
This is super helpful in test code: https://github.com/iov-one/iov-core/pull/1352/files